cmake/Dependencies.cmake

```diff
@@ -1324,6 +1324,55 @@ if(USE_ROCM AND ROCM_VERSION_DEV VERSION_LESS "5.2.0")
   include_directories(SYSTEM ${THRUST_PATH})
 endif()
 
+# ---[ XPU
+if(USE_XPU)
+  include(${CMAKE_CURRENT_LIST_DIR}/public/xpu.cmake)
+  # Support GCC only in the moment.
+  if(PYTORCH_FOUND_XPU AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+    # # -- Host flags (SYCL_CXX_FLAGS)
+    # Combine with torch_compile_options
+    list(APPEND SYCL_HOST_FLAGS -fPIC)
+    list(APPEND SYCL_HOST_FLAGS -std=c++17)
+    # SYCL headers warnings
+    list(APPEND SYCL_HOST_FLAGS -Wno-deprecated-declarations)
+    list(APPEND SYCL_HOST_FLAGS -Wno-attributes)
+
+    if(CMAKE_BUILD_TYPE MATCHES Debug)
+       list(APPEND SYCL_HOST_FLAGS -g)
+       list(APPEND SYCL_HOST_FLAGS -O0)
+    endif(CMAKE_BUILD_TYPE MATCHES Debug)
```

Comment:
How about `RelWithDebugInfo`?

Reply:
We plan to start from basic building type, like release and debug. With the growing of XPU backend usage, will add in future.

Reply:
Please add a TODO here then.